jupyter_book/commands/__init__.py

```diff
@@ -67,37 +75,58 @@ def build(path_book, path_output, config, toc, warningiserror, build):
     if config is not None:
         book_config["yaml_config_path"] = str(config)
         config_yaml = yaml.safe_load(config.read_text())
-
         # Pop the extra extensions since we need to append, not replace
         extra_extensions = config_yaml.pop("sphinx", {}).get("extra_extensions")
+        # Support Top Level config Passthrough
+        # https://www.sphinx-doc.org/en/latest/usage/configuration.html#project-information
+        sphinx_options = ["project", "author", "copyright"]
```

Comment:
why do we need this here instead of in the `yaml.py` loading script that overwrites the config?

Reply:
without this -- the `pdf` author fields weren't being populated. I suspect this is because when sphinx app gets initialised the pdf config `latex_documents` gets formed and the `yaml` pass through currently setup in `toc.py` doesn't take affect. 

Having this here adds it to `config` when `sphinx.app` get's initialised

Reply:
I think some more work is needed here and I will raise an issue. It would be good to have a convention to setup options and pass `sphinx` options through and non-sphinx ones etc. (as you suggest below)